.github/workflows/test.yml

```diff
@@ -40,7 +44,12 @@ jobs:
           firefox-version: 'latest'
 
       - name: Check out repo
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
+
+      - name: Initialize CodeQL
```

Comment:
Should we ignore test files?

Reply:
Why? Test files are not causing any warnings or errors after enabling code scanning.

Reply:
It seems they are.. It found some security issues in test files, isn't it?

Reply:
No, I don't see test files in the results. Link?

Reply:
@strophy My understanding is that *.spec.js files are typically test files. For example: https://github.com/dashpay/platform/blob/f3eb1716bd9c7c3ba476eed8536e19ae493dae8b/packages/wallet-lib/src/types/Account/methods/decrypt.spec.js#L20-L31

Reply:
Thanks, I've excluded these files

Reply:
It seems we need to exclude `/scripts` as well

Reply:
`scripts/**` excluded, two CodeQL issues remaining to resolve